nidm/nidm-experiment/terms/catalog-v001.xml

```diff
@@ -2,13 +2,11 @@
 <catalog prefer="public" xmlns="urn:oasis:names:tc:entity:xmlns:xml:catalog">
     <uri id="User Entered Import Resolution" name="http://www.w3.org/ns/prov-o-20130430" uri="../imports/prov-o"/>
     <uri id="User Entered Import Resolution" name="http://purl.org/nidash/nidm/ontoneurolog_instruments_import.owl" uri="../imports/ontoneurolog_instruments_import.ttl"/>
-    <uri id="User Entered Import Resolution" name="http://purl.org/nidash/nidm/ontoneurolog_instruments_import.ttl" uri="../imports/ontoneurolog_instruments_import.ttl"/>
     <uri id="User Entered Import Resolution" name="http://purl.org/nidash/nidm/nfo_import.owl" uri="../imports/nfo_import.ttl"/>
     <uri id="User Entered Import Resolution" name="http://purl.org/nidash/nidm/nlx_import.owl" uri="../imports/nlx_import.ttl"/>
     <uri id="User Entered Import Resolution" name="http://purl.org/nidash/nidm/sio_import.owl" uri="../imports/sio_import.ttl"/>
-    <uri id="User Entered Import Resolution" name="http://purl.org/nidash/nidm/qibo_import.ttl" uri="../imports/qibo_import.ttl"/>
+    <uri id="User Entered Import Resolution" name="http://purl.org/nidash/nidm/qibo_import.owl" uri="../imports/qibo_import.ttl"/>
```

Comment:
The import in `nidm-experiment.owl` also needs to be updates to replace `qibo_import.ttl` by `qibo_import.owl`.